packages/expo/src/Notifications/Notifications.ts

```diff
@@ -437,4 +440,28 @@ export default {
     }
     return ExponentNotifications.setBadgeNumberAsync(number);
   },
+
+  async scheduleNotificationWithCalendarAsync(
+    notification: LocalNotification,
+    options: {
+      year?: number;
+      month?: number;
+      hour?: number;
+      minute?: number;
+      second?: number;
+      weakDay?: number;
+      repeat?: boolean;
+    } = {}
+  ): Promise<string> {
+    return ExponentNotifications.scheduleNotificationWithCalendar(notification, options);
+  },
+
+  async schedulNotificationWithTimerAsync(  notification: LocalNotification,
+    options: {
+      interval?: number;
+      repeat?: boolean;
+    } = {}
+  ): Promise<string> {
+    return ExponentNotifications.scheduleNotificationWithTimer(notification, options);
```

Comment:
https://developer.apple.com/documentation/usernotifications/untimeintervalnotificationtrigger?language=objc

Reply:
I meant that after merging this PR I would be able to call this method from JS on iOS and since there is no `if Platform.OS === android`, JS would actually try to call `ExponentNotifications.scheduleNotificationWithTimer`, which would result in an error cause there is no such method on `EXNotifications` module